applicationinsights.ts

```diff
@@ -207,10 +208,11 @@ export class Configuration {
      * @param value if true dependencies will be correlated with requests
      * @returns {Configuration} this class
      */
-    public static setAutoDependencyCorrelation(value: boolean) {
+    public static setAutoDependencyCorrelation(value: boolean, forceClsHooked?: boolean) {
```

Comment:
I wouldn't tie our public API surface to the specific 3rd party library providing correlation (which we could otherwise swap out at will). Consider using something like `useAsyncHooks` instead.

Reply:
updated to `useAsyncHooks`